<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IndirectDataReduction</class>
 <widget class="QMainWindow" name="IndirectDataReduction">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Indirect Data Reduction</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_20">
    <property name="topMargin">
     <number>9</number>
    </property>
    <item>
     <widget class="QGroupBox" name="gbC2EInstrument">
      <property name="title">
       <string>Instrument</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_27">
       <property name="topMargin">
        <number>1</number>
       </property>
       <item>
        <widget class="MantidQt::MantidWidgets::IndirectInstrumentConfig" name="iicInstrumentConfiguration">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="techniques" stdset="0">
          <stringlist>
           <string>TOF Indirect Geometry Spectroscopy</string>
          </stringlist>
         </property>
         <property name="facility" stdset="0">
          <string>ISIS</string>
         </property>
         <property name="enableDiffraction" stdset="0">
          <bool>false</bool>
         </property>
         <property name="showInstrumentLabel" stdset="0">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QTabWidget" name="twIDRTabs">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <property name="iconSize">
       <size>
        <width>14</width>
        <height>14</height>
       </size>
      </property>
      <widget class="QWidget" name="tabEnergyTransfer">
       <attribute name="title">
        <string>Energy Transfer</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QScrollArea" name="saEnergyTransfer">
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="loEnergyTransfer">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>100</width>
             <height>100</height>
            </rect>
           </property>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabCalibration">
       <attribute name="title">
        <string>Calibration</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QScrollArea" name="saCalibration">
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="loCalibration">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>100</width>
             <height>100</height>
            </rect>
           </property>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabDiagnostics">
       <attribute name="title">
        <string>Diagnostics</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QScrollArea" name="saDiagnostics">
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="loDiagnostics">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>100</width>
             <height>100</height>
            </rect>
           </property>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabTransmission">
       <attribute name="title">
        <string>Transmission</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QScrollArea" name="saTransmission">
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="loTransmission">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>100</width>
             <height>100</height>
            </rect>
           </property>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabSymmertrise">
       <attribute name="title">
        <string>Symmetrise</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <widget class="QScrollArea" name="saSymmetrise">
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="loSymmetrise">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>100</width>
             <height>100</height>
            </rect>
           </property>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabSofQW">
       <attribute name="title">
        <string>S(Q, w)</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_7">
        <item>
         <widget class="QScrollArea" name="saSofQW">
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="loSofQW">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>100</width>
             <height>100</height>
            </rect>
           </property>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabMoments">
       <attribute name="title">
        <string>Moments</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <item>
         <widget class="QScrollArea" name="saMoments">
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="loMoments">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>100</width>
             <height>100</height>
            </rect>
           </property>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QPushButton" name="pbHelp">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>25</width>
          <height>25</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Open interface help page in your web browser.</string>
        </property>
        <property name="text">
         <string>?</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pbPythonExport">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>25</width>
          <height>40</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Export a Python script for the algorithms run on this tab.</string>
        </property>
        <property name="text">
         <string>Py</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_1">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>30</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pbRun">
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>180</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Run conversion to energy process.</string>
        </property>
        <property name="text">
         <string>Run</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_5">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pbManageDirectories">
        <property name="text">
         <string>Manage Directories</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MantidQt::MantidWidgets::IndirectInstrumentConfig</class>
   <extends>QComboBox</extends>
   <header>MantidQtMantidWidgets/IndirectInstrumentConfig.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>twIDRTabs</tabstop>
  <tabstop>pbHelp</tabstop>
  <tabstop>pbPythonExport</tabstop>
  <tabstop>pbRun</tabstop>
  <tabstop>pbManageDirectories</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
